applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp

```diff
@@ -79,10 +79,10 @@ namespace Kratos
 		Vector nodal_deformation = ZeroVector(msElementSize);
 		this->GetValuesVector(nodal_deformation);
 		rRightHandSideVector = ZeroVector(msElementSize);
-		rRightHandSideVector -= prod(left_hand_side_matrix, nodal_deformation);
+		noalias(rRightHandSideVector) -= prod(left_hand_side_matrix, nodal_deformation);

```

Comment:
just to observe that you could have avoided the initialization to zero by just doing

     noalias(rRightHandSideVector) =  -prod(left_hand_side_matrix, nodal_deformation);